themes/classic/modules/ps_shoppingcart/modal.tpl

```diff
@@ -56,12 +56,20 @@
               {else}
                 <p class="cart-products-count">{l s='There is %product_count% item in your cart.' sprintf=['%product_count%' =>$cart.products_count] d='Shop.Theme.Checkout'}</p>
               {/if}
-              <p><strong>{l s='Total products:' d='Shop.Theme.Checkout'}</strong>&nbsp;{$cart.subtotals.products.value}</p>
-              <p><strong>{l s='Total shipping:' d='Shop.Theme.Checkout'}</strong>&nbsp;{$cart.subtotals.shipping.value} {hook h='displayCheckoutSubtotalDetails' subtotal=$cart.subtotals.shipping}</p>
+              <p><span><strong>{l s='Total products:' d='Shop.Theme.Checkout'}</strong></span>&nbsp;<span><strong>{$cart.subtotals.products.value}</strong></span></p>
+              <p><span>{l s='Total shipping:' d='Shop.Theme.Checkout'}</span>&nbsp;<span><strong>{$cart.subtotals.shipping.value} {hook h='displayCheckoutSubtotalDetails' subtotal=$cart.subtotals.shipping}</strong></span></p>
+
+              {if $cart.totals.total_including_tax.value != $cart.totals.total_excluding_tax.value && $cart.totals.total.value == $cart.totals.total_excluding_tax.value}
```

Comment:
What does this line check exactly?

Reply:
It's to check if customer settings is defined on excluded tax, on Front it will display total tax included and if you disable tax on your shop on Front it will only display total with no tax included

Reply:
After #13135 is merged you'll be able to use `$configuration.display_prices_tax_incl`